packages/system/backupstrategy-controller/Makefile

```
@@ -4,7 +4,7 @@ NAMESPACE=cozy-backup-controller
 include ../../../hack/common-envs.mk
 include ../../../hack/package.mk
 
-image: image-backupstrategy-controller
+image: image-backupstrategy-controller image-redis-backup
```

checkmake (0.3.2)

[warning] Target "image" should be declared PHONY.

(phonydeclared)